tb/arith/carrychain_inc/RunAllTests.pro

```diff
@@ -0,0 +1,9 @@
+# =============================================================================
+# Authors:
+#	Gustavo Martin
+# =============================================================================
```

Comment:
Same as my finding in #33, please use a full file header including license part in new `*.pro` files.